### ISSUE TITLE
Improve https://guides.emberjs.com/v2.4.0/routing/defining-your-routes/#toc_wildcard-globbing-routes

### ISSUE THREAD

**Issue by locks** [emberjs/guides#1333]
Mention how to access the wildcard segment.

**Comment by acorncom** [comment]
@locks Could you provide a little more detail here so we can have other folks help here?

**Comment by jeradg** [comment]
You mean something like:

**Comment by jeradg** [comment]
You mean something like:

```js
// app/router.js
Router.map(function() {
  this.route('page-not-found', { path: '/*wildcard' });
});

// app/routes/page-not-found.js
export default Ember.Route.extend({
  model(params) {                  
    return params;                 
  }                                
});                                

// app/templates/page-not-found.hbs

<p>
  Path '{{model.wildcard}}' not found.
</p>
```

**Comment by jeradg** [comment]
(And yes, that's me volunteering to make a PR.)

**Comment by locks** [comment]
@jeradg waiting for that PR ;)

**Comment by yawboakye** [comment]
@locks as mentioned on Slack I'm taking over. I'll submit the PR.

**Comment by acorncom** [comment]
Fixed in #1741
